srtcore/api.cpp

```diff
@@ -2341,7 +2341,7 @@ CUDT* CUDT::getUDTHandle(UDTSOCKET u)
    {
       return s_UDTUnited.lookup(u);
    }
-   catch (std::exception& ee)
+   catch (...)
```

Comment:
I think it's fine to catch catch (CUDTException e) {} as you proposed especially if other functions already follow this convention. I just checked how UDT handles this case and it handle all exceptions. You should answer yourself several questions: what should you do in case if an exception occurred ? Does it matter what exception type it was ?
If you think you should handle exception in some particular way or add the same fix to another function - go ahead.